relation_engine/taxa/rdp/loaders/rdp_taxa_delta_loader.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python
+
+# TODO TEST
+
+import argparse
+import getpass
+import gzip
+import os
+import unicodedata
+from contextlib import ExitStack
+from arango import ArangoClient
+from urllib.parse import urlparse
+
+from relation_engine.taxa.rdp.parsers import RDPNodeProvider
+from relation_engine.taxa.rdp.parsers import RDPEdgeProvider
+from relation_engine.batchload.delta_load import load_graph_delta
+from relation_engine.batchload.time_travelling_database import ArangoBatchTimeTravellingDB
+
+# TODO probably should make some sort of general arg parser since they're all so similar
+
+_LOAD_NAMESPACE = 'rdp_taxa'
+
+def parse_args():
+    parser = argparse.ArgumentParser(description=
```

Comment:
If you want to clean up this indentation, you could do something like
```
desc = (
    "Load one or more RDP taxonomy dump files into an ArangoDB time travelling\n"
    "database, calculating and applying the changes between the prior load and\n"
    "the current load, and retaining the prior load.")
```

Reply:
Fixed, didn't actually know you could do this